python/ray/_private/external_storage.py

```diff
@@ -251,7 +251,12 @@ class FileSystemStorage(ExternalStorage):
             spill objects doesn't exist.
     """
 
-    def __init__(self, directory_path, buffer_size=None):
+    def __init__(
+        self,
+        directory_path: Union[str, List[str]],
+        node_id: Optional[str] = None,
+        buffer_size: Optional[int] = None,
+    ):
         # -- sub directory name --
         self._spill_dir_name = DEFAULT_OBJECT_PREFIX
```

Comment:
Seems we don't need a class member here, it's only used locally inside the constructor.

Reply:
Yeah I kept the original code, but it's not necessary. Cleaned up.